lib/utils.go

```diff
@@ -31,9 +31,11 @@ import (
 
 const zeroHash = "0x0000000000000000000000000000000000000000000000000000000000000000"
 
+var testChainDir = filepath.Join(TestDataDir, TestNetworkNames[params.StatusChainNetworkID])
```

Comment:
I believe the network is chosen from cli, isn't it?

Reply:
Hmmm... I think StatusChain is fixed here in the nodeConfig (the below from develop branch):
```
var nodeConfigJSON = `{
	"NetworkId": ` + strconv.Itoa(params.StatusChainNetworkID) + `,
	"DataDir": "` + TestDataDir + `",
...
```

Reply:
Yeah, it seems I missed this change somewhere. I hope it works since the first attempt.